priv/repo/structure.sql

```diff
@@ -526,6 +526,7 @@ CREATE TABLE `respondents` (
   KEY `respondents_updated_at_index` (`updated_at`),
   KEY `respondents_sanitized_phone_number_index` (`sanitized_phone_number`),
   KEY `respondents_retry_stat_id_fkey` (`retry_stat_id`),
+  KEY `respondents_canonical_phone_number_index` (`canonical_phone_number`),
```

Comment:
Are the changes in this file `structure.sql` related to the update?

Reply:
No, I suppose it was we missed from a previous pull request :disappointed:

Reply:
Namely #2020